manifests/v/v2rayA/v2rayA/2.0.1-1/v2rayA.v2rayA.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: v2rayA.v2rayA
+PackageVersion: 2.0.1-1

```

```suggestion
PackageVersion: 2.0.1
```
Must match control panel due to the already existing versions